hyperspy/signal_tools.py

```diff
@@ -950,6 +952,8 @@ def find(self, back=False):
                     minimum),
                 self.signal.axes_manager.signal_axes[0].index2value(
                     maximum))
+            self.signal.axes_manager.indices = self.coordinates[self.index]
```

Comment:
Why are these two new lines necessay?

Reply:
Yes one of the two is not necessary, some left over... I moved the second line at a more appropriate spot in the `update_plot()` method and this one is indeed necessary, because the pointer was not updating! Possibly since the split of the UI.